src/components/common/buttons/CustomIcon.js

```diff
@@ -4,7 +4,7 @@ import { View } from 'react-native'
 import Icon from '../view/Icon'
 import { withStyles } from '../../../lib/styles'
 
-const CustomIcon = ({ styles, theme, name, color, size, circle = 16, reverse, reverseColor }) => {
+const CustomIcon = ({ styles, theme, name, color, size, circle = 16, reverse, reverseColor, customColor }) => {
```

Comment:
actually I think we need to revise props of this component

reverse means bg and fg changed vice versa ? but it actually doesn't works this way...
let's keep only 2 props: color (for bg color of view) and textColor (or iconColor) for the icon glyph color itself

Reply:
If I interpreted correctly I think I have made the change